ls1intum/ArTEMiS
# Feature/add categories and difficulty to exercises

### Checklist

### Motivation and Context
Categories and exercise levels were not present in the current administration.

### Description
I added a category selector and a difficulty selector to the administration of exercises. Also displaying this information in the newly created student dashboard pages

### Steps for Testing
1. Log in to ArTEMiS
2. Navigate to Course Administration
3. Create a new exercise of each type
4. add categories for each exercise
5. select an exercise difficulty 
6. check if the correct values are displayed in the UI

### Screenshots
![image](https://user-images.githubusercontent.com/10998002/54347755-edef0e00-4647-11e9-80e2-3c6324332373.png)
![image](https://user-images.githubusercontent.com/10998002/54347780-fb0bfd00-4647-11e9-8758-45728fcee627.png)
![image](https://user-images.githubusercontent.com/10998002/54347802-06f7bf00-4648-11e9-81b1-698b00064ce0.png)
